doc/whatsnew/fragments/7208.bugfix

```diff
@@ -0,0 +1,10 @@
+The ``accept-no-raise-doc`` option related to ``missing-raises-doc`` will now
```

Comment:
@DudeNr33 @Pierre-Sassoulas Just thinking out loud: can we create a `breaking_change` category in `towncrier` so this fragments gets automatically placed at the top?
We don't need to call it breaking change as I don't think this actually is, so perhaps `user_action_required` or something. I think that would help avoid confusion.

Reply:
Yeah I think it would be very valuable to have a synthesis of change that user could need to do when upgrading. (for thing like ``no-self-use`` becoming optional and suddenly raising conf warning message, or a bug being fixed that means the behavior will change unexpectedly)

Reply:
Does `towncrier` support something like that out of the box?

Reply:
I checked the `towncrier` issues and it looks like it was supported in the past, but the latest version seems to have issues (see https://github.com/twisted/towncrier/issues/437).
Will look into it if it works with the version we currently use.